src/content/subgames.cpp

```diff
@@ -73,8 +80,10 @@ struct GameFindPath
 			path(path), user_specific(user_specific)
 	{
 	}
+	GameFindPath() { }
```

Comment:
Is this constructor necessary?

Reply:
It seems to be. Unordered map requires it. Try commenting out this line, compiler errors out.